app/services/stomp-websocket.js

```diff
@@ -70,7 +75,17 @@ export default Service.extend({
 
     let onStompConnect = this.makeStompConnectHandler(stompClient, data, handlers, context);
     let onStompError = this.makeStompErrorHandler(handlers, context);
+
+    this.set('client', stompClient);
     stompClient.connect({ }, onStompConnect, onStompError);
     return stompClient;
```

Comment:
Since storing stompClient in this service, we don't need to return it the caller any more.